src/App.tsx

```diff
@@ -21,6 +22,7 @@ function App() {
 						<PublicRoute path="/login" exact component={LoginPage} />
 						<PublicRoute path="/verify" exact component={VerifyEmailPage} />
 						<ProtectedRoute path="/account" exact component={AccountSettingsPage} />
+						<ProtectedRoute path="/event/:id?" exact component={EventPage}/>
```

Comment:
IMO this route should be 
```suggestion
						<ProtectedRoute path="/events/:id" exact component={EventPage}/>
```

Since this is convention for specifying a single item of a collection in a HTTP route, the plural `/events` contains all routes, and `/events/:id` indicates a specific event